wandb/sdk/launch/utils.py

```diff
@@ -114,14 +126,19 @@ def create_project_from_spec(run_spec, api):
     else:
         name = "{}_{}_launch".format(project, run_id)  # default naming scheme
 
+    overrides = run_spec.get("overrides", {})
+    if "entry_point" not in overrides:
+        wandb.termlog("Entry point for project not specified, defaulting to main.py")
```

Comment:
I'm pretty sure we default to whatever was used to run the original script.

Reply:
ah you're right we handle that elsewhere, fixing